### PR TITLE
Functionality: Add priorities

### LINKED ISSUE #19: Define priorities for functionality
@avan1235 will need your help on Android.

### PR DESCRIPTION
**Pull request by kantoniak**
Closes #19.
I assigned 1 or 2 for required elements and 3 or 4 for optional ones. I'll wait for at least 2 approvals.

@avan1235 I added sorting 😊

### CODE REVIEW

File: specs/functionality.md [hunk 1]
```diff
@@ -1,115 +1,129 @@
...
-            * Display predicted occupance (optional)
-            * Has specified food - based on tags (optional)
-            * List menu options (see below)
-            * List historical menus (see below)
-        * Filter canteen list
-            * Open/closed
-            * Minimum rating
-            * Has tags (optional)
-        * Find nearby canteens based on used location (optional on Android)
+            * [2] Open/closed
```
Comment (avan1235) [inline specs/functionality.md]: based on hour in phone and the opening hours from owner so can be [1]

File: specs/functionality.md [hunk 2]
```diff
@@ -1,115 +1,129 @@
...
+            * [1] Live number of visitors (relatively to the full capacity)
+            * [1] List menu options (see below)
+            * [3] Create visitor count widget for mobile devices (optional)
+            * [3] Display predicted occupance (optional)
+            * [3] Has specified food - based on tags (optional)
+            * [3] List historical menus (see below)
+        * [2] Filter canteen list
+            * [2] Open/closed
```
Comment (avan1235) [inline specs/functionality.md]: can also be changed to [1]

File: specs/functionality.md [hunk 3]
```diff
@@ -1,115 +1,129 @@
...
-        * Display menu options
-        * Rate option
-        * Get nutrition values (optional)
-    * View most of the statistics data
-        * Plot cafeteria occupancy
-            * Break by year/month/week/day/hour
-            * Correlated to specified menu options (optional)
-        * Plot menu options prices changes (optional)
-        * Plot different cafeterias occupancy comparision
+        * [1] Display menu options
```
Comment (avan1235) [inline specs/functionality.md]: we can change it to "display menu option with price (and optionally average review result)"

File: specs/functionality.md [hunk 4]
```diff
@@ -1,115 +1,129 @@
...
+    * View most of the statistic data
+        * Plot relative cafeteria occupancy
+            * [1] Break by year/month/week/day/hour
+            * [4] Correlate to specified menu options (optional)
+        * [2] Plot different cafeterias occupancy comparison
             * Break by year/month/week/day/hour
-    * Provide English version (optional)
-    * Implement accessibility features (optional)
-6. Other prediction pipelines (optional)
+        * [3] Plot menu options prices changes (optional)
```
Comment (avan1235) [inline specs/functionality.md]: I would change it to [2] and make it maybe not optional. The more maybe simple but elegant elements we have the better our app will look like

File: specs/functionality.md (hunk 1 above)
Comment (kantoniak) [reply specs/functionality.md]: Opening hours are priority 3 right now. I'll make both 2.

File: specs/functionality.md (hunk 2 above)
Comment (kantoniak) [reply specs/functionality.md]: Same case as above, will make them 2 so they are not optional.

File: specs/functionality.md (hunk 3 above)
Comment (kantoniak) [reply specs/functionality.md]: Will do.

File: specs/functionality.md (hunk 4 above)
Comment (kantoniak) [reply specs/functionality.md]: Yeah, but let's not make everything required. Optional means we can still do it :)

**Comment by kantoniak** [comment]
@Rhantolq @jakubpw could any of you make a second review before I merge? It would be nice to hear more voices :)

**Pull request merged**